recipes/zziplib/all/conanfile.py

```diff
@@ -51,13 +53,25 @@ def layout(self):
     def requirements(self):
         self.requires("zlib/[>=1.2.11 <2]")
 
+    def validate(self):
+        # Old versions do not support llvm >= 15
+        if (Version(self.version) < "0.13.78"
+                and self.settings.compiler in ("clang", "apple-clang")
+                and Version(self.settings.compiler.version) >= "15.0"):
+            raise ConanInvalidConfiguration(f"{self.ref} does not support {self.settings.compiler} >= 15 due to incompatible pointer to integer conversion errors")
```

Comment:
I think we can leave the `{self.ref}` out? if I remember correctly it is reported by conan before printing it, and any programmatic access is tied to the very reference, so it is redundant?